frontend/src/components/AgencyProfile/RetailRescue.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import edit from "./imgs/edit-icon.png";
+
+/**
+ * Functional component for the retail rescue category
+ *
+ * @param {*} agency
+ * @returns {*} Retail Rescue component
+ */
+function RetailRescue({ agency }) {
+  /**
+   * Function returns a string that contains data corresponding
+   * the the given day if the day's bool is true
+   *
+   * @param {*} day
+   * @returns {string} String of start time and location
+   */
+  function displayDay(day) {
+    let displayStr = "";
+    if (day[0]) {
+      displayStr = `${day[1].substring(11, 16)}, ${day[2]}`;
+    } else {
+      displayStr = "N/A";
+    }
+    return displayStr;
+  }
```

Comment:
Why not have 3 different parameters: `selected`, `startTime`, and `location`, instead of an array with multiple data types?